_distro_map.yml

```diff
@@ -20,7 +20,7 @@ openshift-online:
   site_url: https://docs.openshift.com/
   branches:
     online:
-      name: Latest
+      name: '3 Developer Preview'
```

Comment:
BTW, the pretty build links I provided show "Branch Build" in the breadcrumb because it was built from a feature branch, but once landed on the `online` branch, this is what will show up.